### ISSUE TITLE
Household Mailing List Button Not Working in Console App

### ISSUE THREAD

**Issue by jesslopez-sf** [SalesforceFoundation/Cumulus#4018]
When the Household Mailing List button is clicked while in a Lightning Console App, the report is not filtering properly. This is happening as a result of the way we construct the URL for the report. On Line [120 of the HH_CampaignDedupeBTN_CTRL class](https://github.com/SalesforceFoundation/Cumulus/blob/master/src/classes/HH_CampaignDedupeBTN_CTRL.cls#L120), we are checking to see if the User is in Lightning ("Theme4d" -- but the console is actually "Theme4u"). If so, we construct the URL one way. If not, we construct it for Classic. 

```
if (USER_UserService.isUserRunningLightning()) {
                    newPageUrl = USER_UserService.getLightningBaseURL(campaignMailingListReportId) + '?fv0=' + ActiveID;
                } else {
                    newPageUrl = '/' + campaignMailingListReportId + '?retURL=/' + ActiveID + '&pv0=' + ActiveID;
                }
```

USER_UserService's isUserRunningLightning method determines if a User is in Lightning by looking for UserInfo.getUiThemeDisplayed(). If it returns Theme4d, they are:
```

public static Boolean isUserRunningLightning() {
        return (UserInfo.getUiThemeDisplayed() == 'Theme4d');
    }
```

**The issue is that the Console app returns _Theme4u_ and not Theme4d, so we attempt to construct the Classic URL and the report just doesn't work.**

---

### **Steps to Repeat.**
1. Create 2 Contacts

2. Create a Campaign and add the two Contacts as Campaign Members.

3. Go to the Campaign and click the Household Mailing List button (note: if you're in a scratch org with unmanaged code, you'll need to create a new button with no namespace and add that button to your page layout instead).

4. Results: the report loads properly.

5. Create a Console App:

- Go to Setup --> App Manager. 
- New Lightning App
- Name = My Console Test
- Choose Console Navigation in App Options
- Add Campaigns, Contacts, Accounts to Navigation Items.
- Add the System Admin Profile to User Profiles
- Save.

6. Switch to the My Console Test app

7. Go to your Campaign and click your Household Mailing List button. The report shows 0 results.

**Comment by jesslopez-sf** [comment]
**Lurch: Add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fR29QAE" target="_blank">W-030943</a>

**Comment by coriobriensfdo** [comment]
This also happens in regular Lightning (not the console app). 

Also happening on the V4S Volunteer Roster button. Workaround is to use Classic or manually edit the report with the Campaign ID.

**Comment by coriobriensfdo** [comment]
[Another thread.](https://powerofus.force.com/s/question/0D51E00004pvdNOSAY/broken-campaign-household-mailing-list)

**Comment by mrbelvedere** [comment]
Included in beta release 3.157 (Beta 16)

**Comment by mrbelvedere** [comment]
Included in production release 3.157
